tests/modules/mcuboot/external_flash/sysbuild.conf

```diff
@@ -5,3 +5,9 @@
 #
 
 SB_CONFIG_BOOTLOADER_MCUBOOT=y
+SB_CONFIG_NETCORE_EMPTY=y
```

Comment:
the netcore options would go in Kconfig.sysbuild, and only be enabled for nrf5340dk, not nrf52840dk